internal/apis/kfd/v1alpha2/eks/create/kubernetes.go

```diff
@@ -186,7 +197,38 @@ func (k *Kubernetes) Exec() error {
 	return nil
 }
 
-func (k *Kubernetes) copyFromTemplate() error {
+func (*Kubernetes) getCommonDataFromDistribution(furyctlCfg template.Config) (map[any]any, []any, error) {
```

Comment:
we could move this code to template.Config perhaps?

Reply:
I'm not a big fan, while the input is from this package, the logic inside the function is tied to the schema and may or may not change with other providers/schema versions.